<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDefaultQcApp</class>
 <widget class="BaseDlg" name="QDefaultQcApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1540</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QDefaultQcApp</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>质控品名称</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>90</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>1441</width>
     <height>581</height>
    </rect>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>140</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>45</number>
   </attribute>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>1247</x>
     <y>761</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1377</x>
     <y>761</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>1440</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>733</y>
     <width>1440</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>1</width>
     <height>584</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_2">
   <property name="geometry">
    <rect>
     <x>1489</x>
     <y>150</y>
     <width>1</width>
     <height>584</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
